namespace.go

```diff
@@ -141,7 +138,7 @@ func NamespaceWithEnvironmentBinding(name string) NamespaceOption {
 	return func(ns *Namespace) error {
 		provider, err := aad.NewJWTProvider(
 			aad.JWTProviderWithEnvironmentVars(),
-			aad.JWTProviderWithResourceURI(serviceBusResourceURI),
+			aad.JWTProviderWithResourceURI(ns.ServiceBusResourceURI),
```

Comment:
What happens if `ns.ServiceBusResourceURI` is the empty string?  Assuming it's broken, we should fall back to the public cloud URI in this case to preserve the old behavior.